### PR TITLE
Change poll sdk and add voting sdk

### PR DESCRIPTION
**Pull request by Szymx95**

### CODE REVIEW

File: content/docs/guides/vote_poll_sdk/dapp_creation/02_connect_wallet.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import { Poll } from './components/Poll'
+import { GlobalStyle, TopBar } from '@waku/vote-poll-sdk-react-components'
+import pollingIcon from './assets/images/pollingIcon.png'
+import { orangeTheme } from '@waku/vote-poll-sdk-react-components/dist/esm/src/style/themes'
+import ReactDOM from 'react-dom'
+import { BrowserRouter, useLocation } from 'react-router-dom'
+import { Route, Switch } from 'react-router'
+import { useWeb3Connect } from './hooks/useWeb3Connect'
+
+const TOKEN_ADDRESS = '0x744d70FDBE2Ba4CF95131626614a1763DF805B9E'
```
Comment (D4nte) [inline content/docs/guides/vote_poll_sdk/dapp_creation/02_connect_wallet.md]: Can you please add a comment to clarify what is that? I assume it should be any ERC20 token contract they wish to use?

File: content/docs/guides/vote_poll_sdk/dapp_creation/02_connect_wallet.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+const MULTICALL_ADDRESS = '0xeefba1e63905ef1d7acba5a8513c70307c1ce441'
```
Comment (D4nte) [inline content/docs/guides/vote_poll_sdk/dapp_creation/02_connect_wallet.md:121]: I guess this is ropsten's multical address? can you mention it so they now to change it for mainnet.

File: content/docs/guides/vote_poll_sdk/dapp_creation/03_connect_walle_useDapp.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Connect to the Ethereum Wallet
+title: Connect to the Ethereum Wallet useDapp
```
Comment (D4nte) [inline content/docs/guides/vote_poll_sdk/dapp_creation/03_connect_walle_useDapp.md:2]: `title: Connect to the Ethereum Wallet with useDapp`
The title shows in the side bar so best if it's a complete sentence.

File: content/docs/guides/vote_poll_sdk/dapp_creation/_index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Poll/Vote dapp creation
+date: 2022-01-03T11:00:00+1100
+weight: 1
+---
+
+# How to Use the WakuConnect Poll/Vote SDK
```
Comment (D4nte) [inline content/docs/guides/vote_poll_sdk/dapp_creation/_index.md]: It's `Waku Connect` with a space.

File: content/docs/guides/vote_poll_sdk/dapp_creation/_index.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+This part is the same for both Poll and Vote SDK.
+
+To demonstrate how to use the WakuConnect Poll/Vote SDK in your dApp, we will create a TypeScript React app from scratch.
+
+You can then adapt the steps depending on your dApp configuration and build setup.
+
+Only token holders can create & answer polls.
```
Comment (D4nte) [inline content/docs/guides/vote_poll_sdk/dapp_creation/_index.md]: This guide is for both vote and poll SDK, rights? So maybe change to:
`The poll and vote SDK features can only be used by token holders, you must pass the ERC20 token contract of your choice when using the SDK.`

**Comment by D4nte** [comment]
I rebased your changes on latest `origin/develop` and pushed to `origin/szymx95-change-poll-sdk-guide` to help. Feel free to take it.

**Comment by D4nte** [comment]
Please check that `hugo server` can run:

```
▶ hugo server
Start building sites … 
hugo v0.91.2+extended linux/amd64 BuildDate=unknown
ERROR 2022/01/24 13:44:14 [en] REF_NOT_FOUND: Ref "./03_create-a-poll_button": "/home/froyer/src/vacp2p/docs.wakuconnect.dev/content/docs/guides/vote_poll_sdk/dapp_creation/03_connect_walle_useDapp.md:210:1": page not found
Error: Error building site: logged 1 error(s)
Built in 228 ms
```

**Review by D4nte — COMMENTED** [review]
Looks good, few comments.

**Comment by Szymx95** [comment]
> Please check that `hugo server` can run:
> 
> ```
> ▶ hugo server
> Start building sites … 
> hugo v0.91.2+extended linux/amd64 BuildDate=unknown
> ERROR 2022/01/24 13:44:14 [en] REF_NOT_FOUND: Ref "./03_create-a-poll_button": "/home/froyer/src/vacp2p/docs.wakuconnect.dev/content/docs/guides/vote_poll_sdk/dapp_creation/03_connect_walle_useDapp.md:210:1": page not found
> Error: Error building site: logged 1 error(s)
> Built in 228 ms
> ```

Due to company policy I can't really install hugo on my work pc without permission so if you could check if `hugo server` runs it would be awesome.

**Comment by D4nte** [comment]
> > Please check that `hugo server` can run:
> > ```
> > ▶ hugo server
> > Start building sites … 
> > hugo v0.91.2+extended linux/amd64 BuildDate=unknown
> > ERROR 2022/01/24 13:44:14 [en] REF_NOT_FOUND: Ref "./03_create-a-poll_button": "/home/froyer/src/vacp2p/docs.wakuconnect.dev/content/docs/guides/vote_poll_sdk/dapp_creation/03_connect_walle_useDapp.md:210:1": page not found
> > Error: Error building site: logged 1 error(s)
> > Built in 228 ms
> > ```
> 
> Due to company policy I can't really install hugo on my work pc without permission so if you could check if `hugo server` runs it would be awesome.

Yes it runs ok now.

**Pull request merged**